examples/c/7_series/gnss_ins/README.md

```diff
@@ -69,22 +86,82 @@ The example implements custom communication handlers:
 - **Receive Function**: `mip_interface_user_recv_from_device()` manages incoming data
 - **Timeout Handling**: Configurable timeouts for reliable communication
 
+## Filter Data Output
+
+The example streams the following filter data:
+
+### TOW Data
+- **Units**: seconds
+- **Description**: Time of Week - GPS time reference
+- **Format**: Floating-point timestamp value
+
+### Position LLH Data
+- **Units**:
+    - Latitude: degrees
+    - Longitude: degrees
+    - Ellipsoid Height: meters
+- **Description**: Position in Latitude, Longitude, Height coordinate system
+- **Format**: [Latitude, Longitude, Height] vector
+
+### Velocity NED Data
+- **Units**: m/s (meters per second)
+- **Description**: Velocity in North, East, Down coordinate frame
+- **Format**: [North, East, Down] velocity vector
+
+### Euler Angles Data
+- **Units**: radians
+- **Description**: Orientation expressed as Euler angles
+- **Format**: [Roll, Pitch, Yaw] angle vector
+
+## Streaming Output Format
+
+The example displays filter data in the following format:
+```
+TOW = 123456.789    Position LLH = [ 4.123456, -83.54321,  123.4567]    Velocity NED = [ 1.234567, -0.987654,  0.123456]     Euler Angles = [ 0.012345, -0.067890,  1.234567]
+```
+
 ## Usage
 
-1. Connect your 7-series device to the specified serial port
+1. Connect your 7-series GNSS/INS device to the specified serial port
 2. Update the `PORT_NAME` constant if using a different port
-3. Compile and run the example
-4. The program will:
+3. Update the `WHEELED_VEHICLE_APPLICATION` constant if using a wheeled-vehicle application
```

Comment:
Also worth noting that the user will need to update the antenna offsets and how to measure them. We definitely have documentation for this somewhere in the manual, and I would make sure to link to that and possibly stressing how important this is for dual antenna applications.